### PR TITLE
Add edxapp folder.

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Add edxapp folder and its description.

### CODE REVIEW

File: eox_wrapper/edxapp/__init__.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    This file doesn't wrap all the original modules inside the application, just the necessary or the
+    valid contributions.
+
+    Example file:
+
+    backend_route = settings.EOX_WRAPPER_DJANGO_APP
+
+    try:
+        backend = import_module(backend_function)
```
Comment (morenol) [inline eox_wrapper/edxapp/__init__.py:28]: Is backend_function  backend_route?

File: eox_wrapper/edxapp/__init__.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    except ImportError:
+        backend = import_module('eox_wrapper.reliance.django_app')
```
Comment (morenol) [inline eox_wrapper/edxapp/__init__.py:30]: What is eox_wrapper.reliance.django_app ?

File: eox_wrapper/edxapp/__init__.py (hunk 1 above)
Comment (andrey-canon) [reply eox_wrapper/edxapp/__init__.py]: fixed

File: eox_wrapper/edxapp/__init__.py (hunk 2 above)
Comment (andrey-canon) [reply eox_wrapper/edxapp/__init__.py]: @morenol  please check this https://github.com/eduNEXT/eox-wrapper/pull/4

Comment (morenol) [reply eox_wrapper/edxapp/__init__.py]: LGTM, I am just not sure if reliance is the best name for that module

Comment (andrey-canon) [reply eox_wrapper/edxapp/__init__.py]: @morenol  any suggestion ?

Comment (morenol) [reply eox_wrapper/edxapp/__init__.py]: mock, mocks?

**Pull request merged**